src/frontend/screens/Library/components/SteamInstall/SteamInstallDialog.tsx

```diff
@@ -27,21 +43,28 @@ const SteamInstallDialog: React.FC<SteamInstallDialogProps> = ({
         <ul>
           <li>
             {t(
-              'If you wish to continue, be patient since it can take a few minutes to finish depending on your internet connection and system configuration;',
-              'If you wish to continue, be patient since it can take a few minutes to finish depending on your internet connection and system configuration;'
+              'Recommended Specs for a better experience',
+              'Recommended Specs for a better experience'
             )}
+            :
           </li>
+          <ul>
+            <li>
+              {t('Apple Silicon M1 or above', 'Apple Silicon M1 or above')}
+            </li>
+            <li>{t('macOS 14.0 or above', 'macOS 14.0 or above')}</li>
+            <li>{t('16GB of RAM', '16GB of RAM')}</li>
+          </ul>
           <li>
             {t(
-              'HyperPlay will download and start the Steam Setup for you. Please follow Steam instructions on their installer;',
-              'HyperPlay will download and start the Steam Setup for you. Please follow Steam instructions on their installer;'
+              'Be aware that not all games will work and even if they do, they may not run as expected.',
+              'Be aware that not all games will work and even if they do, they may not run as expected.'
             )}
           </li>
           <li>
-            {t('At the end make sure to', 'At the end make sure to')}{' '}
-            <strong>{t("UNCHECK 'Run Steam'", "UNCHECK 'Run Steam'")}</strong>,{' '}
             {t(
-              'since it will block HyperPlay configuration until Steam is closed;'
+              'If you wish to continue, be patient since it can take a few minutes to finish depending on your internet connection and system configuration;',
+              'If you wish to continue, be patient since it can take a few minutes to finish depending on your internet connection and system configuration;'
```

Comment:
```suggestion
              'If you wish to continue, please be patient since it can take a few minutes to finish depending on your internet connection and system configuration.',
              'If you wish to continue, please be patient since it can take a few minutes to finish depending on your internet connection and system configuration.'
```